pyproject.toml

```diff
@@ -64,8 +64,12 @@ test = [
     "opentelemetry-sdk",
     "opentelemetry-exporter-otlp-proto-http",
     "tiktoken",
+    "sqlite_vec",
```

Comment:
why are these dependencies added? we need to figure out how we should think about this group of deps. so I think what we should do is:
- each distro should have a dependency group in our `pyproject.toml` (this was something @raghotham had suggested a long time ago but we could not achieve it due to various torch cpu constraints) 
- then the test group is just any _additional_ dependency you need for running tests
- when you are running integration tests you should be running in the context of a distro anyway so you can do appropriate `uv run --with` clauses to get the required distro deps

Reply:
Sounds good, I've removed the deps commit and will work on this in a separate PR. Thanks!

Reply:
https://github.com/meta-llama/llama-stack/pull/1468